src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const [isClicked, setIsClicked] = useState('🤍');
+  const toggleClicked =() => {
```

Comment:
Remember that we have passed down a function that is supposed to handle this for us. If we are having to rewrite this function, we aren't effectively lifting state up.